app/controllers/api/v2/notifications_controller.rb

```diff
@@ -0,0 +1,36 @@
+module Api
+  module V2
+    class NotificationsController < ApplicationController
+      include ApiTokenAuthenticatable
+
+      before_action :authenticate!
+
+      def create
+        store_notification.tap do |notification_id|
+          NotificationsWorker.perform_async(notification_id)
+
+          render json: { data: { id: notification_id, type: 'notifications' } }, status: :accepted
+        end
+      end
+
+      private
+
+      def store_notification
+        notification_id = SecureRandom.uuid
+        Rails.cache.write("notification_#{notification_id}", notification_params.to_json, expires_in: 1.hour)
+
+        notification_id
+      end
+
+      def notification_params
```

Comment:
Just wondered if should do some validation on these before storing? ie email format or something?